image-create

```diff
@@ -161,15 +166,18 @@ class MachineBuilder:
 
         partial = os.path.join(data_dir, self.machine.image + ".partial")
 
-        # Copy image via convert, to make it sparse again
-        subprocess.check_call(["qemu-img", "convert", "-c", "-O", "qcow2", self.machine.image_file, partial])
+        if not self.iso:
```

Comment:
"partial" is a really confusing name here. `*.partial` files are used in image-download during the curl'ing, so that you can interrupt and resume a download - and only once it finished, it will be renamed to the final file. This avoids broken (interrupted) files.

However, here it is merely some "temporary new" name for "repacking" the just built image into a fresh qcow.

So my vote would be for renaming `partial` to `rebuild`.

Reply:
Thanks for explaining! That was indeed slightly confusing naming, renamed.